### ISSUE TITLE
redux-devtools-cli open argument doesn't work

### ISSUE THREAD

**Issue by donpinkus** [reduxjs/redux-devtools#454]
After installing `redux-devtools-cli` I run `redux-devtools --open` and get this output:

```
18:05 $ redux-devtools --open
[ReduxDevTools] Start server...
--------------------------------------------------------------------------------

   [Busy] Launching SocketCluster
   [Active] SocketCluster started
            Version: 14.3.3
            Environment: dev
            WebSocket engine: ws
            Port: 8000
            Master PID: 18263
            Worker count: 1
            Broker count: 1

   [Done] Migrations are finished
```

Nothing opens. I tried the open argument with the alternatives as well.

**Comment by knpwrs** [comment]
Are there any updates on the status of this? It would be awesome to not have to write a custom local application to run the dev tools app locally.

**Comment by jbhoosreddy** [comment]
Just checking in to see if this issue can be fixed. Thanks

**Comment by johndturn** [comment]
I'd also like to chime in and ask if this is still a priority or not? 

In the meantime, I've found that (if you're using VSCode as your editor), that the VSCode Redux DevTools extension is a valid workaround for the time being. However, it's not ideal, as the default settings for `host` and `port` are incorrect (typo in spelling `localhost`), and each time you open the devtools window (even in the same Code session), you have to set up the settings. 

Also, I've found that you need to click around a decent amount in the embedded dev tools in order to get the correct data to finally show up. 

Again, it's not ideal, and I'd love to know if there's a better alternative, or if this will be fixed at some point?

**Comment by Methuselah96** [comment]
Resolved with [`redux-devtools-cli@1.0.0-2`](https://github.com/reduxjs/redux-devtools/releases/tag/redux-devtools-cli%401.0.0-2).
